litert/vendors/qualcomm/qnn_manager.cc

```diff
@@ -108,15 +116,53 @@ Expected<absl::Span<const QnnSystemInterface_t*>> LoadSystemProvidersFromLib(
   return absl::MakeSpan(interface_providers, num_providers);
 }
 
+absl::Span<const QnnSaver_Config_t*> GetDefaultSaverConfigs(
+    absl::string_view saver_output_dir) {
+  static std::array<QnnSaver_Config_t, 1> saver_configs;
+  saver_configs[0].option = QNN_SAVER_CONFIG_OPTION_OUTPUT_DIRECTORY;
+  saver_configs[0].outputDirectory = saver_output_dir.data();
+
+  static std::array<const QnnSaver_Config_t*, 2> result = {&saver_configs[0],
+                                                           nullptr};
+  return absl::MakeSpan(result.data(), result.size());
+}
+
+static Qnn_Version_t GetExpectedSaverVersion() {
+  Qnn_Version_t saver_version;
+  saver_version.major = QNN_SAVER_API_VERSION_MAJOR;
+  saver_version.minor = QNN_SAVER_API_VERSION_MINOR;
+  saver_version.patch = QNN_SAVER_API_VERSION_PATCH;
+  return saver_version;
+}
+
 }  // namespace
 
 QnnManager::~QnnManager() = default;
 
+LiteRtStatus SaverInit(SharedLibrary& lib, absl::string_view saver_output_dir) {
```

Comment:
How about starting with a verb like `InitSaver` to align other function naming.
Also, it looks like we can make lib a const reference?
```suggestion
LiteRtStatus InitSaver(const SharedLibrary& lib, absl::string_view saver_output_dir) {
```